src/bootupd.rs

```diff
@@ -82,7 +84,7 @@ enum Opt {
 }
 
 /// A message sent from client to server
-#[derive(Debug, Serialize, Deserialize, StructOpt)]
```

Comment:
@cgwalters I was surprised by this one, and I still can't figure what is the purpose. Am I missing anything?

Reply:
I think having `StructOpt` was just a copy-paste mistake - is that the question?

Reply:
Yes, thanks. I just wasn't sure if there was something else behind that derive being there.